tensorlayer/layers/recurrent.py

```diff
@@ -1524,14 +1519,12 @@ def __init__(
             cell_fn,  #tf.nn.rnn_cell.LSTMCell,
             cell_init_args={'state_is_tuple': True},
             n_hidden=256,
-            initializer=tf.random_uniform_initializer(-0.1, 0.1),
```

Comment:
it is used in Line 1107   `with tf.variable_scope(name, initializer=initializer) as vs:`

Reply:
This is line 1527